root/layout/header/bottom.tt

```diff
@@ -79,8 +82,11 @@
         </ul>
     </li>
     <li class="search">
-        <a href="[% c.uri_for('/search') %]">[% l('Search') %]</a>
+        <span class="menu-header">[% l('Search') %]</span>
         <ul>
+            <li>
+                <a href="[% c.uri_for('/search') %]">[% l('Search') %]</a>
+            </li>
```

Comment:
"Search Data"? This page in general is a bit weird and I'm not sure how to call it (I wonder if people know it exists...)